bigtable-dataflow-parent/bigtable-hbase-beam/src/main/java/com/google/cloud/bigtable/beam/CloudBigtableScanConfiguration.java

```diff
@@ -272,12 +273,11 @@ Builder withTableId(ValueProvider<String> tableId) {
     @Override
     public CloudBigtableScanConfiguration build() {
       if (request == null) {
-        ReadHooks readHooks = new DefaultReadHooks();
-        if (scan == null) {
+        if(scan == null){
           scan = new Scan();
         }
-        ReadRowsRequest.Builder builder = Adapters.SCAN_ADAPTER.adapt(scan, readHooks);
-        request =  StaticValueProvider.of(readHooks.applyPreSendHook(builder.build()));
+        request = new RequestWithScanValueProvider(scan.getStartRow(), scan.getStopRow(),
```

Comment:
There's a `BIgtableExtendedScan` that needs to be considered here.  This approach doesn't seem like it would work.

Reply:
I think that we will need to convert the `Scan` to a `Filter` and `RowSet`, even though we don't have all of the information that `Query` needs.